ddtrace/debugging/_exception/auto_instrument.py

```diff
@@ -25,6 +25,14 @@
 )
 
 
+def _private_debug_exception_tag(name):
+    return "_dd.debug.error.%s" % name
```

Comment:
Just a reminder that we don't have inlining in Python, so function calls add overhead

Reply:
fixed